### PR TITLE
[docs] describe how to use your own tree builder

### PR DESCRIPTION
**Pull request by jameshadfield**
[RTD Preview](https://nextstrain--884.org.readthedocs.build/projects/augur/en/884/faq/skip_augur_tree.html)

### CODE REVIEW

File: docs/faq/skip_augur_tree.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+This guide explains the general approach you can use.
+
+
+Exclude sites
+-------------
+
+Augur tree allows you to mask certain problematic sites before tree-building via  `augur tree --exclude-sites <exclude_file> --alignment <alignment>` , where each line in `<exclude_file>` is a 1-based nucleotide position to mask from the alignment. We can recreate this via the following command:
+
+.. code-block:: bash
+  augur mask --sequences <alignment> --mask <exclude_file> --output <masked_alignment>
```
Comment (joverlee521) [inline docs/faq/skip_augur_tree.rst:17]: This code block isn't rendering in the preview.

Comment (joverlee521) [reply docs/faq/skip_augur_tree.rst]: Fixed in [41e94d6](https://github.com/nextstrain/augur/pull/884/commits/41e94d6eebc76ddbe8d504c1dc05e25134017382)

Comment (jameshadfield) [reply docs/faq/skip_augur_tree.rst]: Thanks @joverlee521. I _really_ don't understand rst :(

Comment (joverlee521) [reply docs/faq/skip_augur_tree.rst]: I've been having a hard time with it too. It's been a lot of trial and error 😥

Comment (tsibley) [reply docs/faq/skip_augur_tree.rst]: To maybe help with understanding, from the [rST primer](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) provided by Sphinx, [on directives](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#directives):

> The directive content follows after a blank line and is indented relative to the directive start or if options are present, by the same amount as the options.

In general if you're confounded by rST, then I'd really recommend reading through the primer and then the [other rST guides from Sphinx](https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html).

**Pull request merged**